roles/result/tasks/fio-samples.yml

```diff
@@ -0,0 +1,96 @@
+    uri:
+      url: "http://{{ elasticsearch.server }}:{{ elasticsearch.port }}/{{ elasticsearch.index }}-result/doc"
+      method: POST
+      status_code: 201
+      body: "{{ lookup('template','fio.json.j2') }}"
+      body_format: json
+
+  # read_csv module was introduced in ansible version 2.8, and so can't be used at the time of
+  # this PR, as there wasn't a stable tag for ansible-operator that was released with ansible
+  # version 2.8.
+  # - name: read the clat log file
+  #   read_csv:
+  #     path: "{{ item.path + '/' + 'fio_clat.1.log' }}"
+  #     fieldnames: time_elapsed, value, data_direction, offset
+  #     delimiter: ', '
+  #   register: clat_log
+  #
+  # - name: template out the clat log file
+  #   template:
+  #     src: fio_clat_csv.json.j2
+  #     dest: "{{results.path}}/{{workload.name}}-{{ uuid }}-fio-clat-output-{{item.time_elapsed}}.json"
+  #   with_items: clat_log.list
+
+  # - name:
+  #   template:
+  #     src: fio_clat.json.j2
+  #     dest: "{{results.path}}/{{workload.name}}-{{ uuid }}-fio-clat-output-{{item.split(', ')[0]}}.json"
+  #   loop: "{{ lookup('file', item.path + '/' + 'fio_clat.1.log').splitlines() }}"
+
+  - name: Index log for clat
+    uri:
+      url: "http://{{ elasticsearch.server }}:{{ elasticsearch.port }}/{{ elasticsearch.index }}-clat/_doc"
+      method: POST
+      status_code: 201
+      body: "{{ lookup('template','fio_clat.json.j2') }}"
+      body_format: json
+    loop: "{{ lookup('file', item.path + '/' + 'fio_clat.1.log').splitlines() }}"
+
+  - name: Index log for bw
+    uri:
+      url: "http://{{ elasticsearch.server }}:{{ elasticsearch.port }}/{{ elasticsearch.index }}-bw/_doc"
+      method: POST
+      status_code: 201
+      body: "{{ lookup('template','fio_bw.json.j2') }}"
+      body_format: json
+    loop: "{{ lookup('file', item.path + '/' + 'fio_bw.1.log').splitlines() }}"
+
+  - name: Index log for iops
+    uri:
+      url: "http://{{ elasticsearch.server }}:{{ elasticsearch.port }}/{{ elasticsearch.index }}-iops/_doc"
+      method: POST
+      status_code: 201
+      body: "{{ lookup('template','fio_iops.json.j2') }}"
+      body_format: json
+    loop: "{{ lookup('file', item.path + '/' + 'fio_iops.1.log').splitlines() }}"
+
+  - name: Index log for lat
+    uri:
+      url: "http://{{ elasticsearch.server }}:{{ elasticsearch.port }}/{{ elasticsearch.index }}-lat/_doc"
+      method: POST
+      status_code: 201
+      body: "{{ lookup('template','fio_lat.json.j2') }}"
+      body_format: json
+    loop: "{{ lookup('file', item.path + '/' + 'fio_lat.1.log').splitlines() }}"
+
+  - name: Index log for slat
+    uri:
+      url: "http://{{ elasticsearch.server }}:{{ elasticsearch.port }}/{{ elasticsearch.index }}-slat/_doc"
+      method: POST
+      status_code: 201
+      body: "{{ lookup('template','fio_slat.json.j2') }}"
+      body_format: json
+    loop: "{{ lookup('file', item.path + '/' + 'fio_slat.1.log').splitlines() }}"
```

Comment:
Are we not going to register the result? ie the status code returned? Just in case we hit failures of indexing?